src/Raven.Server/Smuggler/Documents/SmugglerBase.cs

```diff
@@ -49,14 +49,14 @@ public abstract class SmugglerBase
             ISmugglerDestination destination,
             SystemTime time,
             JsonOperationContext context,
-            DatabaseSmugglerOptionsServerSide options = null,
+            DatabaseSmugglerOptionsServerSide options,
             SmugglerResult result = null,
             Action<IOperationProgress> onProgress = null,
             CancellationToken token = default)
         {
             _databaseName = databaseName ?? throw new ArgumentNullException(nameof(databaseName));
             _source = source;
-            _options = options ?? new DatabaseSmugglerOptionsServerSide();
+            _options = options ?? new DatabaseSmugglerOptionsServerSide(AuthorizationStatus.ValidUser);
```

Comment:
When `option` can be null? We default to a valid user then which might be potential source of another issue.

Reply:
Done